tripleo-quickstart-config/metalkube-nodes.yml

```diff
@@ -29,13 +29,13 @@ node_count: 6
 
 flavors:
   openshift_master:
-    memory: '{{openshift_memory|default(default_memory)}}'
-    disk: '{{openshift_disk|default(default_disk)}}'
-    vcpu: '{{openshift_vcpu|default(default_vcpu)}}'
+    memory: '{{openshift_master_memory|default(default_memory, true)}}'
+    disk: '{{openshift_master_disk|default(default_disk, true)}}'
+    vcpu: '{{openshift_master_vcpu|default(default_vcpu, true)}}'
     extradisks: true
 
   openshift_worker:
-    memory: '{{openshift_memory|default(default_memory)}}'
-    disk: '{{openshift_disk|default(default_disk)}}'
-    vcpu: '{{openshift_vcpu|default(default_vcpu)}}'
+    memory: '{{openshift_worker_memory|default(default_memory, true)}}'
```

Comment:
Is true a good default here?

FWIW I was expecting  us to leave the current variables as they are, and just expose a variable that enables easily passing a copy of this entire file, then folks can hack it however they want.

Reply:
Sorry, I didn't removed the 'true' from the previous commit.